### PR TITLE
Add Saucelabs e2e testing

### PR DESCRIPTION
**Pull request by marcod1419**
In preparation of adding saucelabs e2e in Github Actions, this PR adds the new e2e tests as a local utility. It can be run the same way as the old e2e tests, but does require a .env file with saucelabs credentials.

### CODE REVIEW

File: e2e/commands/checkAccessibility.js [hunk 1]
```diff
@@ -3,10 +3,15 @@ const { rootSelector } = require('../config')
 exports.command = function checkAccessibility() {
   this.initAccessibility().verify.accessibility(rootSelector, {
     verbose: false,
+    timeout: 10000,
     rules: {
       'color-contrast': { enabled: true },
       'skip-link': { enabled: false },
       'heading-order': { enabled: false },
+      'duplicate-id': { enabled: false },
```
Comment (jraff) [inline e2e/commands/checkAccessibility.js]: I've disabled quite a few of these rules, should we look at fixes to remove these disables.

File: e2e/output/components/WebVideo/baseline/internet explorer_11.103.png [hunk 2]
```diff
@@ -0,0 +1 @@
+���rӛ��-
```
Comment (jraff) [inline e2e/output/components/WebVideo/baseline/internet explorer_11.103.png]: ?

File: config/nightwatch.saucelabs.conf.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      desiredCapabilities: {
+        browserName: 'microsoftedge',
+        version: '17',
+        platform: 'windows 10',
+        screenResolution: '1280x960',
+      },
+    },
+    ie11: {
+      desiredCapabilities: {
+        browserName: 'internet explorer',
```
Comment (jraff) [inline config/nightwatch.saucelabs.conf.js]: Can we an underscore here. The screenshot being produced has a space in it.

File: e2e/commands/checkAccessibility.js [hunk 4]
```diff
@@ -3,10 +3,15 @@ const { rootSelector } = require('../config')
 exports.command = function checkAccessibility() {
   this.initAccessibility().verify.accessibility(rootSelector, {
     verbose: false,
+    timeout: 10000,
```
Comment (theetrain) [inline e2e/commands/checkAccessibility.js]: Marco: 'reduce it back to what it was'

Comment (marcod1419) [reply e2e/commands/checkAccessibility.js]: Nvm, this actually might be needed, misinterpreted what it did.

**Comment by theetrain** [comment]
This is ready for QA.

**Pull request merged**